CRM/Core/BAO/CustomGroup.php

```diff
@@ -945,135 +945,6 @@ public static function &getCustomGroupDetail($groupId = NULL, $extends = NULL, $
     return $groupTree;
   }
 
-  /**
-   * @deprecated since 5.71, will be removed around 5.85
-   */
-  public static function &getActiveGroups($entityType, $path, $cidToken = '%%cid%%') {
-    // for Group's
-    $customGroupDAO = new CRM_Core_DAO_CustomGroup();
-
-    // get 'Tab' and 'Tab with table' groups
-    $customGroupDAO->whereAdd("style IN ('Tab', 'Tab with table')");
-    $customGroupDAO->whereAdd("is_active = 1");
-
-    // Emits a noisy deprecation notice
-    self::_addWhereAdd($customGroupDAO, $entityType, $cidToken);
-
-    $groups = [];
-
-    $permissionClause = CRM_Core_Permission::customGroupClause(CRM_Core_Permission::VIEW, NULL, TRUE);
-    $customGroupDAO->whereAdd($permissionClause);
-
-    // order by weight
-    $customGroupDAO->orderBy('weight');
-    $customGroupDAO->find();
-
-    // process each group with menu tab
-    while ($customGroupDAO->fetch()) {
-      $group = [];
-      $group['id'] = $customGroupDAO->id;
-      $group['path'] = $path;
-      $group['title'] = "$customGroupDAO->title";
-      $group['query'] = "reset=1&gid={$customGroupDAO->id}&cid={$cidToken}";
-      $group['extra'] = ['gid' => $customGroupDAO->id];
-      $group['table_name'] = $customGroupDAO->table_name;
-      $group['is_multiple'] = $customGroupDAO->is_multiple;
-      $group['icon'] = $customGroupDAO->icon;
-      $groups[] = $group;
-    }
-
-    return $groups;
-  }
-
-  /**
-   * Unused function.
-   * @deprecated since 5.71 will be removed around 5.85
-   */
-  public static function getTableNameByEntityName($entityType) {
```

Comment:
Called from apachesolr_civiAttachments - but that module appears abandoned (install instructions for 4.5)